cmd/terramate/cli/cloud.go

```diff
@@ -477,30 +457,87 @@ func (c *cli) detectCloudMetadata() {
 	}
 }
 
-func getGithubCommit(ghClient *github.Client, repo string, commitName string) (*github.Commit, error) {
+func getGithubCommit(ghClient *github.Client, owner, repo, commit string) (*github.RepositoryCommit, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), defaultGithubTimeout)
 	defer cancel()
 
-	commit, err := ghClient.Commit(ctx, repo, commitName)
+	rcommit, _, err := ghClient.Repositories.GetCommit(ctx, owner, repo, commit, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	return commit, nil
+	return rcommit, nil
 }
 
-func getGithubPR(ghClient *github.Client, repo string, commitName string) (github.Pulls, error) {
+// returns nil, nil if there was no PR associated with commit
+func getGithubPR(ghClient *github.Client, owner, repo, commit string) (*github.PullRequest, error) {
```

Comment:
Would it make sense move this function (and the other github specific functions: `listGithubChecks`, `listGithubPullReviews` `isGithubPRMerged`) to the `cmd/terramate/cli/github` package?

Reply:
Initially I agreed, but when I tried moving them I noticed that we end up with two `github` packages (our helpers and go-github), and in cloud.go we use both, which is also not great. So for now, I opted to not refactor this further.